frontend/package.json

```diff
@@ -22,9 +25,11 @@
     "postcss": "8.4.30",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-hook-form": "^7.46.2",
     "tailwind-merge": "^1.14.0",
```

Comment:
I was getting errors that this package wasn't being resolved so I quickly added it to the dependencies :)